omnigibson/macros.py

```diff
@@ -55,6 +55,9 @@
 # CANNOT be set at runtime
 gm.GUI_VIEWPORT_ONLY = False
 
+# Whether to use the viewer camera or not
+gm.VIEWER_CAMERA_VISIBLE = True
```

Comment:
Could we rename this? RENDER_VIEWER_CAMERA maybe? It's not visibility that matters but whether or not we render it right (yes ik invisible means it wont get rendered)

Reply:
done